jms/org.wso2.carbon.transport.jms/src/main/java/org/wso2/carbon/transport/jms/callback/CommitOrRollback.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.transport.jms.callback;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.messaging.CarbonCallback;
+import org.wso2.carbon.messaging.CarbonMessage;
+import org.wso2.carbon.transport.jms.utils.JMSConstants;
+
+import javax.jms.JMSException;
+import javax.jms.Session;
+
+/**
+ * Call back used for transacted sessions. To commit or rollback the sessions.
+ */
+public class CommitOrRollback implements CarbonCallback {
```

Comment:
Shall we add a meaningfull class name? Class/variable names should be self explanatory.